### DISCUSSION TITLE
Multiple Sites in one Controller

### DISCUSSION THREAD

**Discussion by cinntech** [charlie-haley/omada_exporter#65] [Q&A]
Is there a way to pull all sites?

**Comment by charlie-haley** [comment]
At the moment, no. It's currently 1 site per exporter. I did this mainly due to the risk of the prometheus endpoint growing too large and being very slow to scrape if you have a lot devices in the site.

The exporter itself is quite lightweight so deploying multiple exporters doesn't add much overhead. I'm not completely against the idea of accepting a comma separated list of sites in the config though (or an `ALL` option)
